views-bom/build.gradle

```diff
@@ -1,15 +1,3 @@
-dependencies {
-    constraints {
-        for (Project p : rootProject.subprojects) {
-            if (p.name == "views-bom" || p.name == 'views') continue
-            api "$p.group:micronaut-$p.name:$p.version"
-        }
-
-        runtime "org.freemarker:freemarker:$freemarkerVersion"
```

Comment:
This bom build was different than the aws bom. Do you know if the right thing is to apply our internal.bom plugin here as well @melix ?

Reply:
Is the advantage of having these dependencies specified is that if a user uses the views bom the don't need to specify the freemarker dependency version?

```
implementation(platform("io.micronaut.views:micronaut-views-bom:$micronautVersion"))
implementation "org.freemarker:freemarker:$freemarkerVersion"
```

Reply:
Yes but the correct usage would be:

```
implementation(platform("io.micronaut.views:micronaut-views-bom:$micronautVersion"))
implementation "org.freemarker:freemarker"
```

Note that if those dependencies are migrated to a version catalog, then you'd don't need to declare them in the BOM module (they would use the `managed-` prefix).

Reply:
I've added the dependencies back to `views-bom/build.gradle` with `api` instead of `runtime`

Reply:
Fixed via https://github.com/micronaut-projects/micronaut-views/pull/331